mm/2s2h/Rando/Logic/Regions/Miscellaneous.cpp

```diff
@@ -0,0 +1,54 @@
+			CHECK(RC_ENEMY_DROP_DODONGO,		CanKillEnemy(ACTOR_EN_DODONGO)		&& CanReachRegions({ RR_TERMINA_FIELD, RR_TERMINA_FIELD_DODONGO_GROTTO })),
+			CHECK(RC_ENEMY_DROP_DRAGONFLY,		CanKillEnemy(ACTOR_EN_GRASSHOPPER)	&& CanReachRegions({ RR_SOUTHERN_SWAMP_SOUTH, RR_WOODFALL, RR_STONE_TOWER_TEMPLE_ENTRANCE, RR_STONE_TOWER_TEMPLE_OUTSIDE_SWITCH_ROOM, RR_WOODFALL_TEMPLE_PRE_BOSS_ROOM, RR_WOODFALL_TEMPLE_MAIN_ROOM_UPPER })),
+			CHECK(RC_ENEMY_DROP_EENO,			CanKillEnemy(ACTOR_EN_SNOWMAN)		&& CanReachRegions({ RR_TERMINA_FIELD, RR_SNOWHEAD_TEMPLE_SNOW_ROOM })),
+			CHECK(RC_ENEMY_DROP_FLYING_POT,		CanKillEnemy(ACTOR_EN_TUBO_TRAP)	&& CanReachRegions({ RR_STONE_TOWER_TEMPLE_INVERTED_UNDER_BRIDGE, RR_SNOWHEAD_TEMPLE_BLOCK_ROOM })),
+			CHECK(RC_ENEMY_DROP_FLOORMASTER,	CanKillEnemy(ACTOR_EN_FLOORMAS)		&& CanReachRegions({ RR_IKANA_CASTLE_FLOORMASTER_ROOM })),
+			CHECK(RC_ENEMY_DROP_FREEZARD,		CanKillEnemy(ACTOR_EN_FZ)			&& CanReachRegions({ RR_SNOWHEAD_TEMPLE_BRIDGE_ROOM_AFTER, RR_SNOWHEAD_TEMPLE_MAP_ROOM_UPPER, RR_SNOWHEAD_TEMPLE_PILLARS_ROOM_UPPER, RR_BENEATH_THE_WELL_FREEZARD_ROOM })),
+			CHECK(RC_ENEMY_DROP_GUAY,			CanKillEnemy(ACTOR_EN_CROW)			&& CanReachRegions({ RR_IKANA_CANYON_UPPER, RR_IKANA_CASTLE_COURTYARD, RR_PATH_TO_GORON_VILLAGE, RR_IKANA_CASTLE_COURTYARD, RR_ZORA_CAPE }) || (CanReachRegions({ RR_MOUNTAIN_VILLAGE, RR_PATH_TO_GORON_VILLAGE }) && RANDO_EVENTS[RE_CLEARED_SNOWHEAD_TEMPLE])),
+			CHECK(RC_ENEMY_DROP_HIPLOOP,		CanKillEnemy(ACTOR_EN_PP)			&& CanReachRegions({ RR_WOODFALL, RR_STONE_TOWER_TEMPLE_INVERTED_WIND_ROOM, RR_STONE_TOWER_TEMPLE_SPIKED_BAR_ROOM_UPPER })),
+			CHECK(RC_ENEMY_DROP_IRON_KNUCKLE,	CanKillEnemy(ACTOR_EN_IK)			&& CanReachRegions({ RR_BENEATH_THE_GRAVEYARD_NIGHT_1_BOSS, RR_BENEATH_THE_GRAVEYARD_NIGHT_2_BOSS, RR_MOON_LINK_TRIAL })),
+			CHECK(RC_ENEMY_DROP_KEESE,			CanKillEnemy(ACTOR_EN_FIREFLY)		&& CanReachRegions({ RR_TERMINA_FIELD, RR_STONE_TOWER_TOP, RR_STONE_TOWER_UPPER, RR_STONE_TOWER_MIDDLE, RR_BENEATH_THE_WELL_LEFT_FIRE_KEESE, RR_BENEATH_THE_WELL_RIGHT_FIRE_KEESE, RR_BENEATH_THE_WELL_FREEZARD_ROOM, RR_BENEATH_THE_GRAVEYARD_NIGHT_2_GRAVE_BEFORE_PIT, RR_SNOWHEAD_NEAR_TEMPLE, RR_PATH_TO_SNOWHEAD_LOWER, RR_PATH_TO_SNOWHEAD_MIDDLE })),
+			CHECK(RC_ENEMY_DROP_LEEVER,			CanKillEnemy(ACTOR_EN_NEO_REEBA)	&& CanReachRegions({ RR_TERMINA_FIELD, RR_GREAT_BAY_COAST, RR_ZORA_CAPE })),
+			CHECK(RC_ENEMY_DROP_LIKE_LIKE,		CanKillEnemy(ACTOR_EN_RR)			&& CanReachRegions({ RR_ZORA_CAPE, RR_GREAT_BAY_COAST })),
+			CHECK(RC_ENEMY_DROP_MAD_SCRUB,		CanKillEnemy(ACTOR_EN_DEKUNUTS)		&& CanReachRegions({ RR_WOODFALL, RR_DEKU_PALACE_INSIDE })),
+			CHECK(RC_ENEMY_DROP_MINI_BABA,		CanKillEnemy(ACTOR_EN_KAREBABA)		&& CanReachRegions({ RR_TERMINA_FIELD, RR_SOUTHERN_SWAMP_NORTH, RR_SOUTHERN_SWAMP_SOUTH, RR_WOODFALL_TEMPLE_MAIN_ROOM, RR_WOODFALL_TEMPLE_MAZE_ROOM })),
```

Comment:
This actor is not in `RR_TERMINA_FIELD`. There is the Bio Baba, which is in `RR_TERMINA_FIELD_BIO_BABA_GROTTO`, but that requires a special case of being Zora. I'm fine with logically ignoring this for now. I don't have any ideas on a clean way of expanding `CanKillEnemy` for subtypes of the same actor right now.
```suggestion
			CHECK(RC_ENEMY_DROP_MINI_BABA,		CanKillEnemy(ACTOR_EN_KAREBABA)		&& CanReachRegions({ RR_SOUTHERN_SWAMP_NORTH, RR_SOUTHERN_SWAMP_SOUTH, RR_WOODFALL_TEMPLE_MAIN_ROOM, RR_WOODFALL_TEMPLE_MAZE_ROOM })),
```

Reply:
Update on this: bio baba is not the same actor as mini baba. Mini baba and wilted baba are the ones that share an actor. The kill conditions are identical between those, so the latter point in the above comment is moot.